### PR TITLE
North West 4 - Musa Yuksel - JavaScript - Week 1 (Extra)

### PR DESCRIPTION
**Pull request by musayuksel**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Musa Yuksel
- Your City: North West
- Your Slack Name: Musa

# Homework Details

- Module: JavaScript
- Week: Week 1

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: extra/1-currency-conversion.js [hunk 1]
```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(pound) {
+  let twoDigit = ((pound * 0.99) * 5.7).toFixed(2);// return string value
```
Comment (MCompton96) [inline extra/1-currency-conversion.js:21]: Good solution!

File: extra/2-piping.js [hunk 2]
```diff
@@ -16,26 +16,30 @@
...
 
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = `${format(multiply(add(startingValue , 10) , 2))}`;
 
 /* BETTER PRACTICE */
```
Comment (MCompton96) [inline extra/2-piping.js:39]: Good Musa!

File: extra/3-magic-8-ball.js [hunk 3]
```diff
@@ -60,8 +67,25 @@ function shakeBall() {
...
+  for(let i =0; i<20; i++){
+    if(answer === answers[i]){
+        answerNumber = i;
+    }
+  }
+  if(answerNumber < 5){
+    return "very positive";
+  } else if(answerNumber >=5 && answerNumber < 10){
+    return "positive";
+  } else if(answerNumber >=10 && answerNumber < 15){
```
Comment (MCompton96) [inline extra/3-magic-8-ball.js:80]: Nice Musa, really clean solution!

Comment (MCompton96) [reply extra/3-magic-8-ball.js:80]: There is a slightly even cleaner way of writing the if and else blocks here using something called ternary operators, if you want to add another stretch goal have a look into these and have a think about how that would look here

File: extra/1-currency-conversion.js (hunk 1 above)
Comment (musayuksel) [reply extra/1-currency-conversion.js:21]: Thank you Matt...

File: extra/2-piping.js (hunk 2 above)
Comment (musayuksel) [reply extra/2-piping.js:39]: Thank you

File: extra/3-magic-8-ball.js (hunk 3 above)
Comment (musayuksel) [reply extra/3-magic-8-ball.js:80]: Thank you for your comment... actually, I was thinking to ask about How can I improve that solution... I will look at the " ternary operators" Thank you

Comment (musayuksel) [reply extra/3-magic-8-ball.js]: Added Ternary operators ...

**Pull request closed without merging**